test-adapter/src/main/scala/org/scalajs/sbttestadapter/ScalaJSTask.scala

```diff
@@ -111,10 +125,7 @@ final class ScalaJSTask private (
 }
 
 object ScalaJSTask {
-  private final class LogElement[T](index: Int,
-      log: Logger => (T => Unit), data: T) {
-    def call(arr: Array[Logger]): Unit = log(arr(index))(data)
-  }
+  type LogElement = (Int, Logger => Unit)
```

Comment:
Why not keeping the class with named values? Simply a
```scala
private final case class LogElement(index: Int, log: Logger => Unit)
```
I see several advantages:

* There's a name to that `Int` value
* It would avoid the weird `x._2(loggers(x._1))`
* It would avoid the automatic tuple insertion involved in `logger(index, l => level(l)(msg))` and `logger(index, _.trace(throwable))`